modules/ROOT/pages/deployment/services/deployment-considerations.adoc

```diff
@@ -0,0 +1,25 @@
+= Deployment Considerations
+:toc: right
+
+:nats-clustering-url: https://docs.nats.io/running-a-nats-service/configuration/clustering
+:nats-helm-url: https://docs.nats.io/running-a-nats-service/nats-kubernetes/helm-chart
+
+:description: When using container orchestration, some considerations should be taken into account when deploying services with respect to scaling.
+
+== Introduction
+
+{description}
+
+== Scaling of Services
+
+Though the majority of services can be scaled by starting them multiple times, this is not always possible for some of them mostly due to technical limitations when targeting High Availability or scaled installations.
+
+* The xref:{s-path}/search.adoc[Search service] cannot be started multiple times. The use in HA or scaled installations is therefore limited. There is, for the time being, no replacement planned.
```

Comment:
```suggestion
* The xref:{s-path}/search.adoc[Search service] cannot be started multiple times. The use in HA or scaled installations is therefore limited.
```
Post a discussion with @micbar and @dragotin, the sentence about the future is removed.